spark/src/main/kotlin/com/adevinta/spark/components/toggles/Switch.md

```diff
@@ -41,6 +41,18 @@ SwitchLabelled(
 ) { Text(text = "Switch On") }
 ```
 
+### Styles
+
+The `Switch` and `SwitchLabelled` accept 7 [ToggleIntent](ToggleIntent.kt)s:
+
+- Primary (default)
+- Secondary
+- Success
+- Alert
+- Danger
+- Info
+- Neutral
```

Comment:
We should change the doc as well
```suggestion
- Main 
- Support
- Accent
- Basic (default)
- Success
- Alert
- Danger
- Info
- Neutral
```

Reply:
Yes, I have corrected this in the second commit